Gems/RobotecSplineTools/Code/Source/Tools/SplineToolsEditorComponent.cpp

```diff
@@ -63,6 +68,40 @@ namespace SplineTools
         ReloadCSVAsset();
     }
 
+    void SplineToolsEditorComponent::SaveCsvAsset()
+    {
+        AZ::SplinePtr splinePtr;
+        LmbrCentral::SplineComponentRequestBus::EventResult(
+            splinePtr, GetEntityId(), &LmbrCentral::SplineComponentRequestBus::Events::GetSpline);
+        auto vertices = splinePtr->GetVertices();
+        using AssetSysReqBus = AzToolsFramework::AssetSystemRequestBus;
+        AZ::Data::AssetInfo sourceAssetInfo;
+        bool ok{ false };
+        AZStd::string watchFolder;
+        AZStd::vector<AZ::Data::AssetInfo> productsAssetInfo;
+
+        AssetSysReqBus::BroadcastResult(
+            ok, &AssetSysReqBus::Events::GetSourceInfoBySourceUUID, m_csvAssetId.m_guid, sourceAssetInfo, watchFolder);
+        const AZ::IO::Path sourcePath = AZ::IO::Path(watchFolder) / AZ::IO::Path(sourceAssetInfo.m_relativePath);
+
+        AZ_Printf("SplineToolsEditorComponent", "Save csv asset to %s", sourcePath.c_str());
+
+        bool result = SetSplinePointsToCsv(sourcePath.c_str(), vertices);
+        if (!result)
+        {
+            AZ_Error("SplineToolsEditorComponent", false, "Vertices not saved");
```

Comment:
Is this _error_ necessary? There are some _warning_ messages triggered in `SetSplinePointsToCsv`.